backend/repositories/match-repository.js

```diff
@@ -22,8 +22,32 @@ const storeMatch = async (match) => {
     try {
         result = await storageContext.insertMatch(match)
 
-        await updateRatingForTeam(winningTeam, match.ratingChange, storageContext)
-        await updateRatingForTeam(losingTeam, -match.ratingChange, storageContext)
+        // FIXME: foosbot's report of rating change is incorrect if the number of players on each side is not the same
```

Comment:
fixme comments don't belong in production code. pls remove comment and create an issue for that